### ISSUE TITLE
childImageSharp for multiple media

### ISSUE THREAD

**Issue by schmanat** [strapi/gatsby-source-strapi#115]
Hi everyone. 

I set up a Gatsby Site with Companies, this content-type use the media type. For the frontend I would like to you gatsby-image but there seems no way to use it. Trying to query (graphql) the childImageSharp I get the following error:

```
error  Cannot query field "childImageSharp" on type "StrapieCompaniesImages".
```

I found a similar issue (#63) but is there no way to query this media type? After changing it to "use a single image" everything works fine. 

How can a achieve it to use Media (multiple) files.

Thanks for your response.

**Comment by rasimack** [comment]
Hi there, 
I still have this problem. I dont find the childImageSharp to query. Someone found a solution?

<img width="236" alt="Screen Shot 2020-05-28 at 8 10 19 pm" src="https://user-images.githubusercontent.com/12076395/83210051-7b206480-a11f-11ea-98eb-c6ef6207717d.png">

**Comment by AbdallahAbis** [comment]
Any solution guys? only the thumbnail is a file, and it's the only one that has childImageSharp 

![graphql](https://i.ibb.co/qj2Hqsc/Screen-Shot-2020-05-30-at-19-43-24.png)

**Comment by denisgoryaynov** [comment]
Hi, I opened a pr to fix the issue: #118, you can check the fork on my github page
